app/views/events/_events_table.erb

```diff
@@ -9,20 +9,22 @@
       <th>Duration</th>
       <th>Category</th>
       <th>Status</th>
+      <th>Template</th>
       <th></th>
     </tr>
   </thead>
 
   <tbody>
     <% events.each do |event| %>
-      <tr>
+      <tr class='<%= e.is_template ? "highlighted" : nil  %>'>
```

Comment:
Should this be `e.is_template`, or `event.is_template` ?

Reply:
@kgf Yes that is correct, I am fixing that now, thanks.

Reply:
Could you reuse the view helper ?